### PR TITLE
Update overview.md

### PR DESCRIPTION
**Pull request by sig-cpotts**
Add new Marketplace location notice and URL.

### CODE REVIEW

**Review by devmehtabd — APPROVED** [review]

File: overview.md [hunk 1]
```diff
@@ -26,16 +28,14 @@ Detect for Azure DevOps is architected to integrate seamlessly with build and re
 
 ## Documentation ##
 
-Instructions and examples are available on our [Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/622618/Synopsys+Detect+for+Azure+DevOps)
-
-For information on the full capabilities of Detect visit [Detect Public Confluence](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Synopsys+Detect)
```
Comment (devmehtabd) [inline overview.md:31]: Why was this removed?

Comment (sig-cpotts) [reply overview.md:31]: @devmehtabd I updated the URL for the ADO instructions and left out the other one since it is the same document.

File: overview.md [hunk 2]
```diff
@@ -26,16 +28,14 @@ Detect for Azure DevOps is architected to integrate seamlessly with build and re
```
Comment (shantyk) [inline overview.md:28]: Minor but do we want a period at the end of line 25?

**Review by shantyk — APPROVED** [review]

**Review by dmamidibd — APPROVED** [review]
